### DISCUSSION TITLE
Error in Stamps Processing Number 6

### DISCUSSION THREAD

**Discussion by Rofiatulumami** [alexisInSAR/EZ-InSAR#100] [Help]
Hi, Excuse me i want to ask regarding the error that i got when running step 6 of stamps processing, how can I resolve this error? thanks. 
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/161690953/cb198144-383a-438c-bde1-4093fc779472)

**Comment by alexisInSAR** [comment]
Hi, 

A StaMPS parameter does not seem correct regarding your data: unwrap_gold_n_win=32

Kind regards, 

Alexis
